Form-Controls/styles.css

```diff
@@ -0,0 +1,70 @@
+{
+    margin:0;
+    padding:0
+}
+.body{
+    background-position: center;
+      background-size: cover;
+      font-family: Georgia, 'Times New Roman', Times, serif;
+      margin-top: 30px;
+      font-size: 15px;
+  }
+  
+   main{
+    display: grid;
+    grid-template-columns: auto;
+    grid-template-rows: auto;
+    justify-content: center;
+    padding: 15px;
+    background-color: blueviolet;
+    text-transform: uppercase;
+    float: inline-start;
+   }
+
+ 
+ h1{
+    background-color: blueviolet ;
+    text-align: center;
+    font-weight: bolder;
+    color: black;
+ }
+ #name{
+    padding: 10px;
+    width: 300px;
+
+ }
+#email{
+    padding: 10px;
+    width: 300px;
+}
+#colors{
+padding: 10px;
+width: 300px;
+}
+#sizes{
+padding: 10px;
+width: 300px;
+}
+ #date{
+    padding: 10px;
+    width: 300px;
+
+}
+button{
+    margin-left: 200px;
```

Comment:
What will happen when the viewport is 280px wide? Try looking at your layout in Devtools devices mode and choosing different devices. 

https://developer.chrome.com/docs/devtools/device-mode/